src/app/components/PropertyDetailSection.tsx

```diff
@@ -113,6 +113,8 @@ const PropertyDetailSection: FC<PropertyDetailSectionProps> = ({
           <div>
             <div className="flex w-full justify-center">
               <Pagination
+                role={undefined}
```

Comment:
I'm not too familiar with the Pagination component we're using so I'm curious, did you have to pass a role to overwrite role in the component?

Reply:
Yes; it provides "navigation" by default.

Reply:
Wow that's silly

Reply:
Agreed.  I wonder if it's worth filing an issue with NextUI, because I agree that the default semantics are incorrect.